extract_unique_sensors_01.py

```diff
@@ -87,3 +87,9 @@ def splitting_extracted_sensors(candidate_sensors, sensor_name):
         sensors_list.append(sensor_)
     return sensors_list
 
+if __name__ == "__main__":
+    sensors_data = read_sensors(file_name, url)
+    extracted_sensors = extract_all_sensors(sensors_data, kat_sensor)
+    unique_sensors_names = splitting_extracted_sensors(extracted_sensors, kat_sensor)
+    
+   
```

Comment:
Too many empty lines.
Aren't you supposed to print out the uniq sensors?

Reply:
Oh yes, I'm supposed to call the print() in the if __name__ condition. I will add that, and remove empty lines.